cg-producer/Dockerfile

```diff
@@ -39,6 +39,8 @@ RUN mkdir /cggen
 # move files
 COPY ./PyCG/ /pycg
 COPY ./entrypoint.py /cggen/entrypoint.py
+COPY ./pycg_producer /cggen/pycg_producer
```

Comment:
Please use dashes (`-`) as separators on directory names instead of low dashes (`_`).

Reply:
Since this directory hosts a Python package, it should follow the [Pep8](https://pep8.org/#package-and-module-names) naming conversions. The package names should have short, all-lowercase names. The underscope is accepted but is [discouraged](https://pep8.org/#package-and-module-names). So, we can rename the directory maybe to `pycgproducer` if you think that it would suit better.

Reply:
Ah, I see. In that case it's better to keep the underscore for readability.